### ISSUE TITLE
Umlauts not shown correctly

### ISSUE THREAD

**Issue by webratte** [deltachat/deltachat-desktop#541]
- Platform (linux/mac/windows): LinuKubuntux/
- Delta Chat Desktop Version: 0.98.1-preview
- Expected behavior: Show umlauts (ä,ü,ö)
- Actual behavior: There are a code is shown instead the expected umlaut
- Steps to reproduce the problem: Add a contact with umlauts in the name
- Screenshots:

![umlaute](https://user-images.githubusercontent.com/6864851/50733142-3c83b600-1188-11e9-932a-d41be653171c.png)

This only happens in the contact list. In the chat it's shown correctly

**Comment by Simon-Laux** [comment]
Also that `=?utf-8?` Thingie happened to split groups.

**Comment by ralphtheninja** [comment]
I just wonder how to reproduce. Created a new group and it looks fine

![screenshot from 2019-01-06 13-43-02](https://user-images.githubusercontent.com/308049/50736103-16c2d500-11b9-11e9-86e0-55236c7d155a.png)

**Comment by webratte** [comment]
No idea.
I have the desktop client installed just today.

The chats was created with android client a few days ago.

Maybe this could be releated ?

**Comment by webratte** [comment]
And my example is no group.
The missing letter is a "ü".

Let me know if I can help.

**Comment by ralphtheninja** [comment]
> The chats was created with android client a few days ago.

With `android-ii` client or the older version?

**Comment by ralphtheninja** [comment]
> And my example is no group.

Hmm, how can it not be a group? Is the umlaut inside an email address or a name?

**Comment by webratte** [comment]
Sorry, the a fewer ii version.

But the last message was sendet with the current version (I belive 🤔)

**Comment by webratte** [comment]
>  Is the umlaut inside an email address or a name?

In the name.

**Comment by webratte** [comment]
Later, if I have more time, I could you offer the Header of this mail.

But won't post it on public.
Is there a way to send it to you?

**Comment by Simon-Laux** [comment]
we all have delta chat

**Comment by ralphtheninja** [comment]
Managed to reproduce. Tried creating a group from android with the name "ümlaut foo bar"

![screenshot from 2019-01-06 14-05-26](https://user-images.githubusercontent.com/308049/50736301-40313000-11bc-11e9-8440-51a363e7bba0.png)

I also noticed that if I named the group "ümlaut" (at creation time) the group name wasn't transferred or shown in desktop (instead showed the name of the person)

**Comment by webratte** [comment]
> we all have delta chat

And what to do with this information? 😉

**Comment by Simon-Laux** [comment]
> > we all have delta chat
> 
> And what to do with this information? wink

You could send us a message there privately.. But anyways @ralphtheninja already got it reproduced so it shouln't be necessary anymore. Otherwise Our forum has also Private Messaging as far as I know.

**Comment by ralphtheninja** [comment]
The chat name comes directly from core. Not sure what to do about this.

@r10s Hilfe!

**Comment by r10s** [comment]
@ralphtheninja i think iconv is missing, see https://github.com/deltachat/deltachat-core/issues/468
the core compiles without it, but it cannot convert later ...
(in a train, my network will go away soon ....)

**Comment by Jikstra** [comment]
@r10s i wonder if the iconv thing might be related to the https://github.com/deltachat/deltachat-desktop/issues/527 bug?

**Comment by ralphtheninja** [comment]
I'm still not following why we would need something like `iconv`. Why do we need some conversion if we send utf8?

**Comment by Jikstra** [comment]
@ralphtheninja very good question

**Comment by Simon-Laux** [comment]
I found a way to reproduce it: 
1. do it on desktop
2. make a group with a utf-8 char in the name
that happens:
- The groupname should be displayed correctly on the client/account
- After switching to a different account that is in this group (check on desktop client) you should see that the groupname isn't displayed correctly

TL;DR;
Only happens when the groupname was set by a other account and only happens on desktop as far as I know.

**Comment by Simon-Laux** [comment]
Does this still happen with the rust core? I didn't saw that bug in a long time

**Comment by Simon-Laux** [comment]
I cant reproduce it anymore so I believe it's solved
